app/views/hostgroups/index.html.erb

```diff
@@ -6,6 +6,8 @@
   <thead>
     <tr>
       <th><%= sort :label, :as => _('Hostgroup|Name') %></th>
+      <th><%= sort :hosts_count, :as => _('Hosts') %></th>
```

Comment:
The default sort order for counters should be DESC so when you click it once it's descending, see 846433a and the domains index.